CMakeLists.txt

```
@@ -278,7 +278,12 @@ if(openPMD_USE_INTERNAL_TOML11)
     add_subdirectory("${openPMD_SOURCE_DIR}/share/openPMD/thirdParty/toml11")
     message(STATUS "toml11: Using INTERNAL version '3.7.1'")
 else()
-    find_package(toml11 3.7.1 CONFIG REQUIRED)
+    # Since toml11 4.0 was a breaking change,
+    # it needs separate find_package() calls
+    find_package(toml11 3.7.0 CONFIG QUIET)
```

I think we intentionally had 3.7.1 due to a bug fix in it:
```suggestion
    find_package(toml11 3.7.1 CONFIG QUIET)
```